code/modules/clothing/ears/ears.dm

```diff
@@ -0,0 +1,47 @@
+
+//Ears: currently only used for headsets and earmuffs
+/obj/item/clothing/ears
+	name = "ears"
+	w_class = WEIGHT_CLASS_TINY
+	throwforce = 0
+	slot_flags = SLOT_EARS
+	resistance_flags = 0
+
+/obj/item/clothing/ears/earmuffs
+	name = "earmuffs"
+	desc = "Protects your hearing from loud noises, and quiet ones as well."
+	icon_state = "earmuffs"
+	item_state = "earmuffs"
+	strip_delay = 15
+	equip_delay_other = 25
+	resistance_flags = FLAMMABLE
+
+/obj/item/clothing/ears/earmuffs/Initialize(mapload)
+	..()
+	SET_SECONDARY_FLAG(src, BANG_PROTECT)
+	SET_SECONDARY_FLAG(src, HEALS_EARS)
+
+/obj/item/clothing/ears/headphones
+	name = "headphones"
+	desc = "Unce unce unce unce. Boop!"
+	icon = 'icons/obj/clothing/accessories.dmi'
+	icon_state = "headphones_off"
+	item_state = "headphones_off"
+	slot_flags = SLOT_EARS | SLOT_HEAD | SLOT_NECK		//Fluff item, put it whereever you want!
+	actions_types = list(/datum/action/item_action/toggle_headphones)
+	var/icon_state_on = "headphones_on"
+	var/icon_state_off = "headphones_off"
+	var/item_state_on = "headphones_on"
+	var/item_state_off = "headphones_off"
+	var/headphones_on = FALSE
+
+/obj/item/clothing/ears/headphones/ui_action_click(owner, action)
+	headphones_on = !headphones_on
+	if(headphones_on)

```

Comment:
Change the initial icon/item state to `headphones`, then you can do:
`item_state = "[initial(icon_state)]_[headphones_on ? "on" : "off"]"` for both icon and item states. And then you don't have to declare 4 new variables for this.
Also move the icon operations to `update_icon` and call it on initialize